documentation/assemblies/managing/assembly-tuning-config.adoc

```diff
@@ -23,7 +23,7 @@ The following tools help with Kafka tuning:
 
 * xref:cruise-control-concepts-str[Cruise Control] generates optimization proposals that you can use to assess and implement a cluster rebalance
 * xref:proc-setting-broker-limits-str[Kafka Static Quota plugin] sets limits on brokers
-* xref:type-Rack-reference[Rack configuration] spreads broker partitions across racks
+* xref:type-Rack-reference[Rack configuration] spreads broker partitions across racks and allows consumers to fetch data from the nearest replica
```

Comment:
The Rack-awareness is just one of many things which need to be achieved to consume from nearest replica. So I think we need to be careful about this to make sure it does not suggest that it is all what is needed.

The distribution of the replicas across the partitions is also significantly more important for reliability and avauilability. So I think we need to make sure it does not get lost in this.

Reply:
> The Rack-awareness is just one of many things which need to be achieved to consume from nearest replica. So I think we need to be careful about this to make sure it does not suggest that it is all what is needed.

@scholzj Should we mention `affinity` rules and `topologySpreadConstraints` here? Or were you thinking of something else?

Reply:
> The distribution of the replicas across the partitions is also significantly more important for reliability and avauilability. So I think we need to make sure it does not get lost in this.

We could add a note under these links:

_If you're considering rack configuration, make sure that topic  replication is set up for high availability. Having replicated topics across brokers, you can use rack awareness to distribute the replicas across racks._

Reply:
TBH, I would be fine to leave it as it was. The chapter it links to has the details. But if you want to change this, then it should be clear to focus on the main use-case for rack awareness which is availability and reliability. The fetching from closest replica is a niche use-case with a lot of gotchas and caveats.

Reply:
> We could add a note under these links:
> _If you're considering rack configuration, make sure that topic replication is set up for high availability. Having replicated topics across brokers, you can use rack awareness to spread partition replicas across racks._

Its not like this is wrong. It is more that I don't feel like it adds too much nd at the end you still need to go to https://strimzi.io/docs/operators/in-development/full/configuring.html#type-Rack-reference and read the whole stuff there.

Reply:
Thanks. I'll leave it as it is.